src/gui/__init__.py

```diff
@@ -14,45 +14,166 @@ def start(self):
+        airline_button.pack(pady=10)
 
-        # Left frame: Client actions (Add, Edit, Delete)
-        left_frame = ctk.CTkFrame(master=middle_frame, width=200)
-        left_frame.pack(pady=10, padx=10, fill="both", expand=True)
+        # New record buttons at bottom of sidebar
+        add_client_button = ctk.CTkButton(master=sidebar_frame, text="+ New Client", width=160, corner_radius=10, command=self.open_new_client_window)
+        add_client_button.pack(pady=20, side="bottom")
 
-        add_button_client = ctk.CTkButton(master=left_frame, text="Add Client", command=self.add_client)
-        add_button_client.pack(pady=10, fill="x")
+        add_flight_button = ctk.CTkButton(master=sidebar_frame, text="+ New Flight", width=160, corner_radius=10)
+        add_flight_button.pack(pady=10, side="bottom")
 
-        add_button_flight = ctk.CTkButton(master=left_frame, text="Add Flight", command=self.add_flight)
-        add_button_flight.pack(pady=10, fill="x")
+        add_airline_button = ctk.CTkButton(master=sidebar_frame, text="+ New Airline", width=160, corner_radius=10)
+        add_airline_button.pack(pady=10, side="bottom")
 
-        add_button_airline = ctk.CTkButton(master=left_frame, text="Add Airline", command=self.add_airline)
-        add_button_airline.pack(pady=10, fill="x")
+        # Main Content Frame (Right side)
+        content_frame = ctk.CTkFrame(master=app)
+        content_frame.grid(row=0, column=1, sticky="nsew", padx=20, pady=20)
 
-        #Bottom frame: Client Dashboard
-        bottom_frame = ctk.CTkFrame(master=app)
-        bottom_frame.pack(pady=10, padx=10, fill="x")
+        # Search bar
+        search_frame = ctk.CTkFrame(master=content_frame, width=600, height=50)
+        search_frame.pack(fill="x", pady=20)
 
-        #Here we put our client table
-        client_details_textbox = ctk.CTkTextbox(master=bottom_frame, height=500)
-        client_details_textbox.pack(pady=10, fill="x")    
+        search_label = ctk.CTkLabel(master=search_frame, text="Client Dashboard", font=("Arial", 24, "bold"))
+        search_label.pack(anchor="w")
+
+        search_entry = ctk.CTkEntry(master=search_frame, width=400, placeholder_text="Search name or ID...")
+        search_entry.pack(side="left", padx=10, pady=10)
+
+        # Client Table Header
+        table_frame = ctk.CTkFrame(master=content_frame)
+        table_frame.pack(fill="both", expand=True)
+
+        header_frame = ctk.CTkFrame(master=table_frame)
+        header_frame.pack(fill="x", pady=10)
+
+        headers = ["Client ID", "Type", "Name", "Address Line 1", "City", "State", "Zip Code", "Country", "Phone No."]
+        for header in headers:
+            label = ctk.CTkLabel(master=header_frame, text=header, width=10, font=("Arial", 10, "bold"))
+            label.pack(side="left", padx=10)
+
+        # Client Data
+        data_frame = ctk.CTkFrame(master=table_frame)
```

Comment:
There's a bug now where if we add a client it won't be shown in the table unless the app is closed and re-opened again. To fix this we can store this `data_frame` in `self` and then have an update method that removes the table's children and re-adds them by `calling client_repo.list()`